commerce-add-ons/payment-providers/buckaroo/release-notes.md

```diff
@@ -0,0 +1,13 @@
+# Release Notes
+
+In this section, we have summarized the changes to Buckaroo Payment Provider for Commerce released in each version. Each version is presented with a link to the [Commerce Buckaroo Payment Provider issue tracker](https://github.com/umbraco/Umbraco.Commerce.PaymentProviders.Buckaroo/issues) showing a list of issues resolved in the release.  We also link to the individual issues themselves from the detail.
+
+If there are any breaking changes or other issues to be aware of when upgrading they are also noted here.
+
+{% hint style="info" %}
+For details of releases for **Buckaroo Payment Provider for Umbraco Commerce**, refer to the [Change log file on Github](../../changelog-archive/buckaroo.md).
+{% endhint %}
```

Comment:
this can be removed as this is the previous release notes for Vendr, however Buckaroo was not part of it so there are no release notes for Vendr
```suggestion
```